pkg/controller/cloudlet_resources.go

```diff
@@ -34,6 +34,7 @@ type CloudletResources struct {
 	platformNodeCount int
 	vms               []edgeproto.VMResource // list of VMs to pass to platform GetClusterAdditionalResources()
 	numVms            int
+	debug             bool // print debug messages during calculations
```

Comment:
This is just to disable some of the spanlogs in normal cases, because it's too chatty. But for unit-testing/etc to debug an issue we may want to re-enable those logs, in which case we can temporarily set debug to true.